gtsfm/utils/metrics.py

```diff
@@ -85,7 +86,7 @@ def compute_rotation_angle_metric(wRi_list: List[Optional[Rot3]], gt_wRi_list: L
     for (wRi, gt_wRi) in zip(wRi_list, gt_wRi_list):
         if wRi is not None and gt_wRi is not None:
             errors.append(comp_utils.compute_relative_rotation_angle(wRi, gt_wRi))
-    return GtsfmMetric("rotation_averaging_angle_deg", errors)
+    return GtsfmMetric("rotation_error_angle_deg", errors)
```

Comment:
renamed the metrics to not be module-specific, but rather to describe the type of error more.

Reply:
So I needed this change so we could use this same error metric for the outputs of both BA and Averaging, without it being called "averaging_xyz" in both places